timesheet_activity_report/__openerp__.py

```diff
@@ -3,7 +3,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     'name': 'Timesheet Activities Report',
-    'version': '8.0.1.0.3',
+    'version': '8.0.1.1.0', #FIXME is it the right version? I assume the module needs to be upgraded...

```

Comment:
Indeed I simplified the SQL view (not for the BR yet, I wanted to do that in a separate PR), you're right. Thanks!